apophis/cmds/remind.py

```diff
@@ -1,25 +1,61 @@
 """ Reminder Command """
 from cmds.command import Command
-# import asyncio
-# import time
+import asyncio
 import json
 import logging
+import time
+from datetime import datetime
 
 
-units_to_seconds = {
-    "minute" : 60, 
-    "hour" : 3600, 
-    "day" : 86400, 
-    "week": 604800, 
-    "month": 2592000
-}
+def convert_times(when):
+    t = {
+        "year": int(datetime.now().strftime("%Y")),
+        "month": int(datetime.now().strftime("%m")),
+        "day": int(datetime.now().strftime("%d")),
+        "hour": int(datetime.now().strftime("%H")),
+        "minute": int(datetime.now().strftime("%M")),
+    }
+
+    iau = [
+        {"int": int(w.split()[0]), "unit": w.split()[1]}
+        for w in when
+    ]
+
+    for i in range(len(iau)):
```

Comment:
`i` is only ever used to index into `iau` suggesting that `for x in iau` is a better choice.